docs/api-reference/layers/text-layer.md

```diff
@@ -124,9 +124,16 @@ The padding of the background, `[padding_x, padding_y]` in pixels.
 
 Specifies a prioritized list of one or more font family names and/or generic family names. Follow the specs for CSS [font-family](https://developer.mozilla.org/en-US/docs/Web/CSS/font-family).
 
-##### `characterSet` (Array | String, optional)
+##### `characterSet` (Array | Set | String, optional)
```

Comment:
Is it worth adding a note to `examples/website/tagmap/tagmap-layer/README.md` also?

Reply:
What kind of note? The example readmes just link to layer documentation.